FAQ.md

```diff
@@ -0,0 +1,251 @@
+6. [Contributing](#contributing)
+7. [Miscellaneous](#miscellaneous)
+
+---
+
+## General Questions
+
+### **Q: What is CDEvents?**
+**A:** CDEvents is an open specification for Continuous Delivery events, designed to promote interoperability between tools in the software production ecosystem. It standardizes the format of events across various phases of the software development lifecycle, enabling seamless communication and integration between different CI/CD tools.
+
+---
+
+### **Q: Why was CDEvents created?**
+**A:** CDEvents was created to address the fragmentation in the Continuous Delivery (CD) toolchain. By providing a common event specification, it allows different tools and platforms to communicate more effectively, reducing integration complexity and preventing vendor lock-in. This fosters a more flexible and adaptable CD pipeline.
+
+---
+
+### **Q: Who is behind CDEvents?**
+**A:** CDEvents is developed and maintained by a collaborative working group comprising industry experts, open-source contributors, and representatives from various organizations committed to enhancing Continuous Delivery practices.
+
+---
+
+### **Q: How does CDEvents differ from other event specifications?**
+**A:** Unlike proprietary event formats or tool-specific integrations, CDEvents is an open, vendor-neutral specification focused solely on Continuous Delivery events. It emphasizes interoperability, flexibility, and ease of adoption across diverse toolchains and environments.
+
+---
+
+## Technical Questions
+
+### **Q: How does CDEvents ensure event consistency across tools?**
+**A:** CDEvents provides a standardized event schema that all compliant tools must follow. This includes consistent naming conventions, required fields, and data formats. By adhering to this schema, events emitted by different tools maintain uniformity, simplifying integration and processing.
+
+---
+
+### **Q: What event types are defined by CDEvents?**
+**A:** CDEvents defines a variety of event types covering different stages of the CD pipeline, including but not limited to:
+- **Build Events:** Indicate the start, success, or failure of a build process.
+- **Test Events:** Capture test execution details and results.
+- **Deploy Events:** Represent deployment actions, including initiation and completion.
+- **Pipeline Events:** Provide insights into the overall workflow and pipeline status.
+- **Approval Events:** Handle manual approval steps within the pipeline.
+- **Artifact Events:** Manage artifact creation, storage, and retrieval.
+
+For a comprehensive list, refer to the [CDEvents specification](https://github.com/cdevents/spec/blob/main/spec.md).
+
+---
+
+### **Q: What technologies does CDEvents use?**
+**A:** CDEvents is technology-agnostic and can be implemented using various protocols and data formats. Common implementations use JSON or YAML for event payloads and can be transported over HTTP, WebSockets, or message brokers like Kafka and RabbitMQ.

```

Comment:
This is completely correct. CDEvents can be implemented using various protocols for transport. 
CDEvents defines a binding for CloudEvents, which supports multiple underlying event protocols, like HTTP, WebSockets, Kafka, RabbitMQ.

In terms of data format, the only supported format is JSON.

Reply:
Changed